src/Teams.Tests/Domain/AnswerTests.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using Teams.Domain;
+
+namespace Teams.Tests.Domain
+{
+    public class AnswerTests
+    {
+        private Guid _predefinedId;
+        private Answer _defaultAnswerAsText;
+        private Answer _defaultAnswerAsOptions;
+
+        [SetUp]
+        public void Setup()
+        {
+            _predefinedId = new Guid("f4340761-1211-48eb-9bea-b6052c28bac3");
```

Comment:
Why do u need this guid?

Reply:
This is the predefined Guid used as a mock guid strictly for testing purposes. Needed it just to make sure that the predefined guid will continue existing.